Python/structmember.c

```diff
@@ -80,7 +81,9 @@ PyMember_GetOne(const char *obj_addr, PyMemberDef *l)
         v = PyUnicode_FromString((char*)addr);
         break;
     case Py_T_CHAR:
-        v = PyUnicode_FromStringAndSize((char*)addr, 1);
+        // case ...: char value = ... requires a C23-compatible compiler
+        value = FT_ATOMIC_LOAD_CHAR_RELAXED(*(char *) addr);
+        v = PyUnicode_FromStringAndSize(&value, 1);
```

Comment:
If you want a local variable in a case statement, declare a scope:

```
case Py_T_CHAR: {
    char value = FT_ATOMIC_LOAD_CHAR_RELAXED(*(char *) addr);
    v = PyUnicode_FromStringAndSize(&value, 1);
}
```

Reply:
Oh didn't know that, thanks!